obs-packaging/wait-obs.sh

```diff
@@ -33,6 +35,8 @@ run_in_docker() {
 # Check all project has finshed the build
 wait_finish_building() {
 	while osc pr -q | grep '(building)'; do sleep 5; done
+	# just in case something goes wrong
+	while osc pr -q | grep '(building)'; do sleep 5; done
```

Comment:
Can you explain why you need to repeat the command?
BTW, you may consider using the `--watch` option.

Also, it may be worth typing out the osc commands and option in extended mode, for better self-documenting code.

Reply:
Also, you may consider adding some backoff mechanism to the `osc pr` calls inside `is_published`

Reply:
That is maybe  something no improve I was paranoic that for some reason the loop exit.